lms/djangoapps/verify_student/tests/factories.py

```diff
@@ -6,10 +6,13 @@
 from verify_student.models import SoftwareSecurePhotoVerification
 
 
+# Factories are self documenting
+# pylint: disable=missing-docstring
```

Comment:
I suspect putting the message at this level will disable the check for the whole file.
```
Is it possible to locally disable a particular message?¶
Yes, this feature has been added in Pylint 0.11. This may be done by adding “#pylint: disable=some-message,another-one” at the desired block level or at the end of the desired line of code
```
http://docs.pylint.org/faq.html#is-it-possible-to-locally-disable-a-particular-message